libexec/scoop-virustotal.ps1

```diff
@@ -29,35 +30,58 @@
 . "$psscriptroot\..\lib\manifest.ps1"
 . "$psscriptroot\..\lib\buckets.ps1"
 . "$psscriptroot\..\lib\json.ps1"
+. "$psscriptroot\..\lib\config.ps1"
 
 reset_aliases
 
-$opt, $apps, $err = getopt $args 'sa:' @('arch=', 'scan')
+$opt, $apps, $err = getopt $args 'a:gs' @('arch=', 'global', 'scan')
 if($err) { "scoop virustotal: $err"; exit 1 }
 $architecture = ensure_architecture ($opt.a + $opt.arch)
+$global = $opt.g -or $opt.global
 
 $_ERR_UNSAFE = 2
 $_ERR_EXCEPTION = 4
 $_ERR_NO_INFO = 8
 
 $exit_code = 0
 
+Function Info($msg) {
```

Comment:
INFO/WARN/ERROR functions should be moved to core.ps1 (we should add a better logging/output system anyway 😁)

Reply:
Agreed.